Automattic/jetpack
# JITM: use the A8c-Mc-Stats package

#### Changes proposed in this Pull Request
Use the A8c-Mc-Stats package to record stats instead of calling Jetpack methods.

We're currently using the `JETPACK__VERSION` in the stat names. Let's remove that since it's not necessary to include the version in the stat id.

#### Jetpack product discussion
* p9dueE-2HL-p2

#### Does this pull request change what data or activity we track or use?
* No

#### Testing instructions:
1. Checkout this branch.
2. Connect Jetpack.
4. Check the `jetpack-jitm/asset_cdn-viewed/` MC stat. 
5. Navigate to Jetpack -> Dashboard.
6. You should see the `asset_cdn` JITM: "Give your site a speed boost".
7. Check the `jetpack-jitm/asset_cdn-viewed/` MC stat. It should have incremented by 1.
